File: cdk_integration_tests/src/python/s3.py
from aws_cdk import App, Stack, aws_s3


class AppStack(Stack):
    def __init__(self, app: App, id: str) -> None:
        super().__init__(app, id)

        bucket = aws_s3.Bucket(
            self,
            "example",
            encryption=aws_s3.BucketEncryption.S3_MANAGED,
        )


app = App()
AppStack(app, "example-stack")
app.synth()

File: cdk_integration_tests/src/typescript/s3.ts
import { App, Stack, StackProps } from "aws-cdk-lib";
import * as s3 from 'aws-cdk-lib/aws-s3';
import { Construct } from 'constructs';

export class exampleStack extends Stack {
  constructor(scope: Construct, id: string, props?: StackProps) {
    super(scope, id, props);

    new s3.Bucket(this, 'example', {
      encryption: s3.BucketEncryption.S3_MANAGED,
    });
  }
}

const app = new App();
new exampleStack(app, 'example-stack');
app.synth();
